corehq/apps/reports/filters/location.py

```diff
@@ -0,0 +1,63 @@
+from dimagi.utils.decorators.memoized import memoized
+from corehq.apps.locations.models import SQLLocation
+from .users import ExpandedMobileWorkerFilter
+from .api import EmwfOptionsView
+from django.core.urlresolvers import reverse
+from django.utils.translation import ugettext_lazy
+from corehq import toggles
+
+
+class LocationGroupFilter(ExpandedMobileWorkerFilter):
```

Comment:
follow up thought - i'm not sure if this is possible/easy but it seems like it'd be nice if that is what it's doing if we could parameterize the existing filter rather than subclassing it. it seems a little high-overhead to have to make a new class and API endpoint every time we want to support a different set of stuff. i'm not too familiar with this part of the code so it might be that this is really hard to implement and we shouldn't bother, just mentioning in case it's easy